src/main/java/usecases/pull_game_ended/PgeInteractor.java

```diff
@@ -0,0 +1,47 @@
+package usecases.pull_game_ended;
+
+import entities.Player;
+
+import java.util.List;
+
+/**
+ * Interactor for the Pull Game Ended Data use case
+ */
+public class PgeInteractor implements PgeInputBoundary {
+
+    private final PgeOutputBoundary presenter;
+
+    /**
+     * Constructor for PgeInteractor
+     * @param presenter Object to call for output
+     */
+    public PgeInteractor(PgeOutputBoundary presenter) {
+        this.presenter = presenter;
+    }
+
+    /**
+     * Method to return the list of player ids from a list of players
+     * @param players contains a list of players
```

Comment:
minor comment but not important: for extra readability, consider using the tab key to evenly space the parameter name and description